api/mysagw/identity/serializers.py

```diff
@@ -59,6 +60,39 @@ class IdentitySerializer(TrackingSerializer):
         "phone_numbers": "mysagw.identity.serializers.PhoneNumberSerializer",
     }
 
+    def validate(self, *args, **kwargs):
+        validated_data = super().validate(*args, **kwargs)
+
+        is_organisation = validated_data.get("is_organisation")
+        organisation_name = validated_data.get("organisation_name")
+        no_organisation_name_msg = (
+            'Can\'t set "is_organisation" without an organisation_name.'
```

Comment:
I usually use single quotes in the inner quote to avoid escaping as it's happening here

Reply:
I prefer it that way. In python there is no difference between single and double quotes. In language there is.